platforms/mobile/ios/fastlane.md

```diff
@@ -0,0 +1,64 @@
+# Fastlane Match Deployment
```

Comment:
I've been using `fastlane pilot upload` to upload WellPoint to Test Flight after building an IPA. This command probably needs mentioning.

Reply:
Hi @dextermb, this is the iOS specific section - where we use an in-IDE button.